examples/Microsoft.DBforPostgreSQL_flexibleServers@2023-06-01-preview/main.tf

```diff
@@ -0,0 +1,79 @@
+terraform {
+  required_providers {
+    azapi = {
+      source = "Azure/azapi"
+    }
+    azurerm = {
+      source = "hashicorp/azurerm"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {
+  }
+}
+
+provider "azapi" {
+  skip_provider_registration = false
+}
+
+variable "resource_name" {
+  type    = string
+  default = "acctest0001"
+}
+
+variable "replica_resource_name" {
```

Comment:
This variable is not used, please remove it.

Reply:
Removed.